rules/coding-style/src/Node/NameImporter.php

```diff
@@ -191,6 +192,13 @@ private function isNamespaceOrUseImportName(Name $name): bool
     private function isFunctionOrConstantImportWithSingleName(Name $name): bool
     {
         $parentNode = $name->getAttribute(AttributeKey::PARENT_NODE);
+
+        $fullName = $name->toCodeString();
+        $autoImportNames = $this->parameterProvider->provideParameter(Option::AUTO_IMPORT_NAMES);
+        if ($autoImportNames && ! $parentNode instanceof Node && Strings::contains($fullName, '\\') && function_exists($fullName)) {
```

Comment:
Use `! Strings::contains($fullName, '\\')` followed previous suggestion:

```suggestion
        if ($autoImportNames && ! $parentNode instanceof Node && ! Strings::contains($fullName, '\\') && function_exists($fullName)) {
```